sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredentialBuilder.java

```diff
@@ -29,7 +31,7 @@ public SharedTokenCacheCredentialBuilder username(String username) {
      *
      * @return An updated instance of this builder.
      */
-    public SharedTokenCacheCredentialBuilder allowUnencryptedCache() {
+    SharedTokenCacheCredentialBuilder allowUnencryptedCache() {
```

Comment:
Could use @schaabs 's opinion but should this be allowed for SharedTokenCacheCredential? Since enablePersistentCache is effectively always true.

Reply:
We discussed offline and have decided to remove the configuration and make the default `True` for this release. Since the `SharedTokenCacheCredential` will not create the cache if it doesn't exist. It will only ever support using the unencrypted cache if the encrypted is not available and the unencrypted already exists.

Reply:
enabled this by default in build method.